src/connectors/traits.rs

```diff
@@ -0,0 +1,33 @@
+use crate::connectors::sink::Sink;
+use crate::errors::Result;
+
```

Comment:
Would be nice to have a `prelude` module that exports all the traits so they can be imported via `*`